frontend/src/components/workspaces/Templates/TemplatesTableRowSettings/TemplatesTableRowSettings.tsx

```diff
@@ -0,0 +1,59 @@
+import { Menu, Dropdown } from 'antd';
+import Button from 'antd-button-color';
+import {
+  PlayCircleOutlined,
+  EditOutlined,
+  DeleteOutlined,
+  SettingOutlined,
+} from '@ant-design/icons';
+
+export interface ITemplatesTableRowSettingsProps {
+  id: string;
+  editTemplate: (id: string) => void;
+  deleteTemplate: (id: string) => void;
+}
+const TemplatesTableRowSettings = ({ ...props }) => {
+  const { id, editTemplate, deleteTemplate } = props;
+
+  return (
+    <Dropdown
+      overlay={
+        <Menu>
+          <Menu.Item
+            className="xs:hidden block"
+            key="1"
+            icon={<PlayCircleOutlined />}
+          >
+            Create
+          </Menu.Item>
+          <Menu.Item
+            key="1"
+            icon={<EditOutlined />}
+            onClick={() => editTemplate(id)}
```

Comment:
The `editTemplate` will trigger some kind of modal right?

Reply:
exactly